### PR TITLE
Make backing type a bool, since it defaults anyway on 1 of 2 options.

### PR DESCRIPTION
**Pull request by dereuromark**
https://github.com/cakephp/bake/pull/968

I would still like to at least propose this for consideration.
Given that there are now only 2 - and string by default, it seems kind of overkill to have a full option for it.
It seems `-i` could suffice?

### CODE REVIEW

**Comment by othercorey** [comment]
Sorry, but I think this is a confusing option. Maybe if it was `--int` or `--string` that would be clear.

**Comment by dereuromark** [comment]
So just --int ?

File: src/Command/EnumCommand.php [hunk 1]
```diff
@@ -83,11 +82,10 @@ public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionPar
...
             'Bake backed enums for use in models.'
-        )->addOption('backing-type', [
-            'help' => 'The return type for the backed enum class',
-            'default' => 'string',
-            'choices' => ['string', 'int'],
-            'short' => 'b',
+        )->addOption('int', [
+            'help' => 'Using backed enums with int instead of string as return type',
+            'boolean' => true,
+            'short' => 'i',
```
Comment (markstory) [inline src/Command/EnumCommand.php:88]: I don't know if we need a short option for this as `int` isn't that long.

Comment (dereuromark) [reply src/Command/EnumCommand.php:88]: Well, it can still be convinient :)

Comment (markstory) [reply src/Command/EnumCommand.php:88]: Fair enough.

**Pull request merged**